Sprint-1/errors/3.js

```diff
@@ -7,3 +7,10 @@ const last4Digits = cardNumber.slice(-4);
 // Then run the code and see what error it gives.
 // Consider: Why does it give this error? Is this what I predicted? If not, what's different?
 // Then try updating the expression last4Digits is assigned to, in order to get the correct value
+ console.log(last4Digits)
+
+/*
+TypeError. cardNumber is not an array or string being slice works with those types. so I get the same result with what I gess before when i run the code
```

Comment:
This works! Can you figure out another way to fix the problem by changing the expression last4Digits is assigned to instead of changing the type of cardNumber?

Reply:
I have updated. by converting the number to string and extracting its last 4 characters instead of change its type. Thank you!